build.gradle

```diff
@@ -54,32 +54,29 @@ repositories {
     }
 }
 
-
 dependencies {
     api platform('com.exactpro.th2:bom:3.2.0')
 
-    implementation 'com.exactpro.th2:grpc-common:3.11.1'
     implementation 'com.exactpro.th2:common:3.40.0'
+    implementation 'com.exactpro.th2:grpc-codec:0.0.2'
 
     implementation 'io.github.microutils:kotlin-logging:2.1.23'
 
-    implementation 'org.slf4j:slf4j-api'
-    implementation 'org.slf4j:slf4j-log4j12'
+    implementation 'org.apache.logging.log4j:log4j-slf4j-impl'
+    implementation 'org.apache.logging.log4j:log4j-1.2-api'
+    implementation 'org.apache.logging.log4j:log4j-api'
 
     implementation 'com.fasterxml.jackson.core:jackson-core'
     implementation 'com.fasterxml.jackson.module:jackson-module-kotlin'
     implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
 
-    implementation 'org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.21'
-    implementation 'org.jetbrains.kotlin:kotlin-reflect:1.6.21'
-
-    implementation('com.github.ajalt:clikt:2.8.0')
+    implementation "com.github.ajalt:clikt:2.8.0"
 
     implementation 'com.google.protobuf:protobuf-java-util'
 
-    testImplementation 'org.junit.jupiter:junit-jupiter:5.9.0'
-    testImplementation 'org.jetbrains.kotlin:kotlin-test-junit:1.7.10'
-    testImplementation 'org.mockito.kotlin:mockito-kotlin:4.0.0'
+    testImplementation "org.junit.jupiter:junit-jupiter:5.9.0"
+    testImplementation "org.jetbrains.kotlin:kotlin-test-junit:${kotlin_version}"
```

Comment:
No needs to use the version. It will be automatically resolved because of the Kotlin plugin